x/epochs/module.go

```diff
@@ -84,7 +84,7 @@ func (AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx client.Context, mux *r
 
 // GetTxCmd returns the capability module's root tx command.
 func (a AppModuleBasic) GetTxCmd() *cobra.Command {
-	return cli.GetTxCmd()
+	return nil
```

Comment:
What is the long term plan for this?

Reply:
wdym, it has no TX command?

Long term plan is that you don't  have to implement this function for interface compat, if you don't use a tx cli

Reply:
Ah, was trying to figure out why we were returning GetTxCmd before, makes sense